### ISSUE TITLE
SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode

### ISSUE THREAD

**Issue by holms** [brainsiq/merge-yaml-cli#5]
Am I something doing wrong in here?

```
Romans-iMac holms$ merge-yaml -i compose.yml compose.prod.env.yml compose.prod.yml cloud.prod.yml -o cloud.yml
/opt/local/lib/node_modules/merge-yaml-cli/index.js:6
class YamlMerger extends EventEmitter {
^^^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/opt/local/lib/node_modules/merge-yaml-cli/cli.js:3:16)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

**Comment by robbieh** [comment]
You need to upgrade to a newer nodejs. That fixed the same problem for me.

**Comment by holms** [comment]
Would be nice if @brainsiq  would mention that in README :)

**Comment by brainsiq** [comment]
I will do. Sorry I missed this 8 months ago!

**Comment by brainsiq** [comment]
You need node version 4 or greater.

**Comment by 422158** [comment]
i have node js in version 4.2.6, still get this error

**Comment by brainsiq** [comment]
Ok, @422158  thanks for listing the version so I can reproduce this. I thought it was because class was not supported but it requires 'use strict' to be added to the module. It's not clear what version is required to not have to do this but I can't see it having any adverse effects anyway!

**Comment by brainsiq** [comment]
I've just published version 1.1.2 with a fix for this issue. Minimum required version of Node.js is 4.
